src/foam/nanos/column_config/ColumnConfigPropertyValue.js

```diff
@@ -0,0 +1,223 @@
+      code: function(of, propNames) {
+        var props = of.getAxiomsByClass(foam.core.Property);
+        var allColumnNames = props.map(p => p.name);
+        if ( ! propNames ) {
+          return props.filter(p => ! p.networkTransient ).map(p => p.name);
+        } props = [];
+        return propNames.filter(n => { 
+          return allColumnNames.includes(n.split('.')[0]) && ! this.returnProperty(of, n).networkTransient;
+        });
+      }
+    },
+    {
+      name: 'returnProperty',
+      documentation: 'returns last property for chain of properties e.g. city property for address.city',
+      type: 'PropertyInfo',
+      args: [
+        {
+          name: 'of',
+          type: 'ClassInfo'
+        },
+        {
+          name: 'propName',
+          class: 'String'
+        }
+      ],
+      code: function(of, propName) {
+        var cls = of;
+        var property;
+        if ( foam.String.isInstance(propName) ) {
+          var propNames = propName.split('.');
+          for ( var i = 0 ; i < propNames.length ; i++ ) {
+            property = foam.String.isInstance(propNames[i])
+            ? cls.getAxiomByName(propNames[i])
+            :  foam.Array.isInstance(propNames[i]) ? 
+            cls.getAxiomByName(propNames[i]) : propNames[i];
+            if ( ! property )
+              break;
+            cls = property.of;
+          }
+        } else
+          property = propName;
+        return property;
+      },
+      javaCode: `
+        ClassInfo ci = of;
+        PropertyInfo p = null;
+        String[] propNames = propName.split("\\\\.");
+        StringBuilder sb = new StringBuilder("find");
+
+        for ( int i = 0 ; i < propNames.length ; i++ ) {
+          if ( ( p == null && i != 0 ) || ci == null )
+            break;
+          p = (PropertyInfo) ci.getAxiomByName(propNames[i]);
+          if ( i != propNames.length - 1 ) {
+            try {
+              Class cls;
+              if ( p instanceof foam.core.AbstractFObjectPropertyInfo ) {
+                cls = p.getValueClass();
+              } else {
+                char[] arr = p.getName().toCharArray();
+                arr[0] = Character.toUpperCase(arr[0]);
+                sb.append(arr);
+                String s = sb.toString();
+                Method m = ci.getObjClass().getMethod(s, foam.core.X.class);
+                cls = m.getReturnType();
+                sb.setLength(4);
+              }
+              ci = (ClassInfo) cls.getMethod("getOwnClassInfo").invoke(null);
+            } catch (Exception e) {
+              Logger logger = (Logger) getX().get("logger");
+              logger.error(e);
+              return null;
+            }
+          }
+        }
+        return p;
+      `
+    },
+    {
+      name: 'returnPropertyAndObject',
+      type: 'foam.nanos.column.ColumnPropertyValue',
+      documentation: `returns property and object on which such property's functions as f or toCSV can be called`,
```

Comment:
No need for ` when a single-line string without interpolation. Just use ' which parses faster.